packages/desktop-gui/src/settings/settings.scss

```diff
@@ -148,6 +148,10 @@
     position: relative;
     left: -15px;
 
+    &:focus {
+      outline: 0;
```

Comment:
While this fixes the goof, it's [worse for accessibility](https://a11yproject.com/posts/never-remove-css-outlines/). I'd try setting a width on `.rc-collapse-header` so it's still there, but doesn't run into "Learn more".

Reply:
I should have known better. Thanks for the info, I will amend that.